workspaces/liferay-paypal-workspace/client-extensions/liferay-paypal-commerce-checkout-step/LCP.json

```diff
@@ -13,18 +19,19 @@
 	"kind": "Deployment",
 	"livenessProbe": {
 		"httpGet": {
-			"path": "/ready",
-			"port": 58081
+			"path": "/",
+			"port": 80
 		}
 	},
 	"loadBalancer": {
-		"targetPort": 58081
+		"cdn": true,
+		"targetPort": 80
 	},
-	"memory": 512,
+	"memory": 50,
 	"readinessProbe": {
 		"httpGet": {
-			"path": "/ready",
-			"port": 58081
+			"path": "/",
+			"port": 80
 		}
```

Comment:
@gamerson ,

If we remove the Spring Controllers, how would we support the following ?

https://github.com/liferay/liferay-portal/blob/c5ede076937e208feae3b423e5e5b54883a68bce/modules/apps/commerce/commerce-client-extension-web/src/main/java/com/liferay/commerce/client/extension/web/internal/util/ClientExtensionCommerceCheckoutStep.java#L115

https://github.com/liferay/liferay-portal/blob/c5ede076937e208feae3b423e5e5b54883a68bce/modules/apps/commerce/commerce-client-extension-web/src/main/java/com/liferay/commerce/client/extension/web/internal/util/ClientExtensionCommerceCheckoutStep.java#L183

Both the sample checkout step and my checkout step are reliant on the Liferay/Server side as part of ClientExtensionCommerceCheckoutStep.

My code is not reliant on an ActionRestController and/or /action being available, but other CX checkout steps might be as seen in our sample checkout step here: https://github.com/liferay/liferay-portal/blob/c5ede076937e208feae3b423e5e5b54883a68bce/workspaces/liferay-sample-workspace/client-extensions/liferay-sample-commerce-checkout-step/src/main/java/com/liferay/sample/ActionRestController.java#L35

Reply:
I tried testing the `catapult.launch` check with the revised ports, but it fails to work even after I change the path to `/ready` etc.